.istanbul.yml

```diff
@@ -0,0 +1,21 @@
+instrumentation:
+  root: apps
+  extensions:
+    - .js
+  default-excludes: true
+  include-all-sources: true
+  excludes: ['**/fields/*.js', '**/index.js', '**/steps.js']
```

Comment:
I'm not sure we can, field and steps are just config, we could possibly test index, make sure the middleware is applied correctly etc?